storybook/stories/Layout/AppHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { AppHeader } from "../../../src/components/Layout/AppHeader";
+import { View } from "react-native";
+import { size } from "../../../src/common/styles";
+import { AppMode } from "../../../src/context/config";
+
+storiesOf("Layout", module).add("AppHeader", () => (
+  <View style={{ margin: size(3) }}>
+    <View style={{ marginBottom: size(1) }}>
+      <AppHeader mode={AppMode.staging} />
+    </View>
+  </View>
+));
```

Comment:
I received render error when trying to look at `<AppHeader/>` storybook. Maybe can put null `<View/>` first if it's not done yet?

Reply:
Oh also, what does it mean by `ATOM` ya?